src/Neo/SmartContract/ApplicationEngine.Runtime.cs

```diff
@@ -407,14 +407,20 @@ protected internal void SendNotification(UInt160 hash, string eventName, Array s
         /// </summary>
         /// <param name="hash">The hash of the specified contract. It can be set to <see langword="null"/> to get all notifications.</param>
         /// <returns>The notifications sent during the execution.</returns>
-        protected internal NotifyEventArgs[] GetNotifications(UInt160 hash)
+        protected internal Array GetNotifications(UInt160 hash)
         {
             IEnumerable<NotifyEventArgs> notifications = Notifications;
             if (hash != null) // must filter by scriptHash
                 notifications = notifications.Where(p => p.ScriptHash == hash);
-            NotifyEventArgs[] array = notifications.ToArray();
+            var array = notifications.ToArray();
             if (array.Length > Limits.MaxStackSize) throw new InvalidOperationException();
-            return array;
+            Array notifyArray = new(ReferenceCounter);
+            foreach (var notify in array)
+            {
+                notifyArray.Add(notify.ToStackItem(ReferenceCounter));
+            }
+
+            return notifyArray;
```

Comment:
this will happens automatically, isn't it?

Reply:
convrt works fine.

Reply:
> this will happens automatically, isn't it?

yes, this code is when i was trying to fkx it. will remove